docs/docs/en/architecture/configuration.md

```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
-│  ├─start-all.sh                       startup all DS services with configurations
-│  ├─stop-all.sh                        shutdown all DS services with configurations
-├─conf                              configurations directory
-│  ├─application-api.properties         API-service config properties
-│  ├─datasource.properties              datasource config properties
-│  ├─zookeeper.properties               ZooKeeper config properties
-│  ├─master.properties                  master-service config properties
-│  ├─worker.properties                  worker-service config properties
-│  ├─quartz.properties                  quartz config properties
-│  ├─common.properties                  common-service [storage] config properties
-│  ├─alert.properties                   alert-service config properties
-│  ├─config                             environment variables config directory
-│      ├─install_config.conf                DS environment variables configuration script [install or start DS]
-│  ├─env                                load environment variables configs script directory
-│      ├─dolphinscheduler_env.sh            load environment variables configs [eg: JAVA_HOME,HADOOP_HOME, HIVE_HOME ...]
-│  ├─org                                mybatis mapper files directory
-│  ├─i18n                               i18n configs directory
-│  ├─logback-api.xml                    API-service log config
-│  ├─logback-master.xml                 master-service log config
-│  ├─logback-worker.xml                 worker-service log config
-│  ├─logback-alert.xml                  alert-service log config
-├─sql                                   .sql files to create or upgrade DS metadata
-│  ├─create                             create SQL scripts directory
-│  ├─upgrade                            upgrade SQL scripts directory
-│  ├─dolphinscheduler_postgre.sql       PostgreSQL database init script
-│  ├─dolphinscheduler_mysql.sql         MySQL database init script
-│  ├─soft_version                       current DS version-id file
-├─script                            DS services deployment, database create or upgrade scripts directory
-│  ├─create-dolphinscheduler.sh         DS database init script
-│  ├─upgrade-dolphinscheduler.sh        DS database upgrade script
-│  ├─monitor-server.sh                  DS monitor-server start script       
-│  ├─scp-hosts.sh                       transfer installation files script                                     
-│  ├─remove-zk-node.sh                  cleanup ZooKeeper caches script       
-├─ui                                front-end web resources directory
-├─lib                               DS .jar dependencies directory
-├─install.sh                        auto-setup DS services script
+├── LICENSE
+│
+├── NOTICE
+│
+├── licenses                                    directory of licenses
+│
+├── bin                                         directory of DS application commands, configrations scripts 
```

Comment:
We should avoid using the shortcut for product name in our document
```suggestion
├── bin                                         directory of DolphinScheduler application commands, configrations scripts 
```

Reply:
same as other keyword below

Reply:
Sure, seems we also need to delete https://github.com/apache/dolphinscheduler/blob/243ee8ba70d50c7d4b251fd43a4c150606014b8b/docs/docs/en/architecture/configuration.md?plain=1#L15

Reply:
I never found it before, we could use it only if we have this sentence in the beginning of our document, mean in https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/about/introduction.html